src/atoms/typography/heading.stories.tsx

```diff
@@ -17,6 +18,13 @@ export const headingKnobs = () => {
 
     return (
         <Heading
+            icon={
+                select(
+                    "icon",
+                    Icons,
+                    Icons.Plus
+                ) as Icons
```

Comment:
Does this need to be cast? Didn't see a cast in another storybook file

Reply:
I copied that from the heading priority right below. I prefer not to have that as well so I removed it for both